appservice/src/deploy/showDeployConfirmation.ts

```diff
@@ -12,16 +12,21 @@ import { delay } from '../utils/delay';
 import { updateWorkspaceSetting } from '../utils/settings';
 import { AppSource, IDeployContext } from './IDeployContext';
 
-export async function showDeployConfirmation(context: IDeployContext, site: ParsedSite, deployCommandId: string): Promise<void> {
+export async function showDeployConfirmation(context: IDeployContext, site: ParsedSite, deployCommandId: string, warningMessages?: string[]): Promise<void> {
     const warning: string = l10n.t('Are you sure you want to deploy to "{0}"? This will overwrite any previous deployment and cannot be undone.', site.fullName);
+    let addedWarnings = ''
+    if (warningMessages) {
+        const warningMessagesString: string = warningMessages.map((message) => `${message}`).join('\n\n');
```

Comment:
This is fine, but one of the rare times that you can use can use Array.reduce 🤣

Reply:
Can't we just do `const warningMessagesString = warningMessages.join('\n\n');`

Reply:
@motm32 do dis

Reply:
I am don't worry